lib/foreman_maintain/error.rb

```diff
@@ -0,0 +1,9 @@
+module ForemanMaintain
+  module Error
+    class Fail < StandardError
+    end
+
+    class Skipped < StandardError
```

Comment:
Since there is not use case for it now, let's remove it, including the related places that deal with it